server/controllers/user/profile.js

```diff
@@ -0,0 +1,33 @@
+const { Work, Review } = require('../../database/models');
+const paramsValidation = require('../../utils/validation/params');
+const { customError } = require('../errors');
+
+const profile = async (req, res, next) => {
+  try {
+    const { id } = await paramsValidation.validateAsync(req.params);
+    const works = await Work.findAll({
+      where: {
+        userId: id,
+      },
+    });
+    const user = await works[0].getUser();
+    const services = await user.getServices();
+    const locations = await user.getLocation();
+    const reviews = await Review.findAll({
```

Comment:
Do we need to get all review we only need to get the top five or ten reviews,
The same for works I think in the first fetch we need to get the most recent four or six works as we have in figma, then when user navigate to the next page we can fetch more

And as we're not getting the whole reviewsm we still need the whole rate for that provider